controllers/jobs.js

```diff
@@ -58,7 +58,7 @@ const deleteJob = async (req, res) => {
   if (!job) {
     throw new NotFoundError(`No job with id ${jobId}`);
   }
-  res.status(StatusCodes.OK).send();
+  res.status(StatusCodes.OK).json({ msg: "The entry was deleted." });
```

Comment:
I'm not sure if this is just from the course but I'd advise against sending a message like this in response to a delete request. A 200 response to a delete request is enough information for the client to know that the resource was deleted, and over time this would just become an unnecessary thing to maintain.